### PR TITLE
Ab#4407 slider component does not strip html tags from short description field

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
- Here's [the card](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Core%20team/Stories/?workitem=4407)
- adds a method that strips html tags to kibble
- test via template is sufficent as StripTags already tested by author (checked with Sarge about approach. Also I cant do any better until I learn golang)

### CODE REVIEW

File: kibble/sample_site/templates/tv/detail.jet [hunk 1]
```diff
@@ -33,3 +33,5 @@ Available To US West: [{{ tvseason.Available.To | zone: "America/Los_Angeles" |
     This is true
     {{ tvseason.CustomFields["string"] }} == {{ tvseason.CustomFields["number"] }}
 {{ end }}
+
+{{ stripHTML("tags <br>should be stripped") }}
```
Comment (additiverich) [inline kibble/sample_site/templates/tv/detail.jet:37]: Just wondering what happens with `stripHTML("1 < 2 && 2 > 1")` or something where it's not an HTML tag but could look conceivably like one

**Comment by gscragg** [comment]
I dont suppose you have tested any of the builtin helpers for cleaning html? https://github.com/CloudyKit/jet/wiki/4.-Built-in-functions#escape-helper

File: kibble/sample_site/templates/tv/detail.jet (hunk 1 above)
Comment (matthewthomsonnz) [reply kibble/sample_site/templates/tv/detail.jet:37]: Works as expected: ` 1 &lt; 2 &amp;&amp; 2 &gt; 1`, I think the author [has it all pretty well covered](https://github.com/grokify/html-strip-tags-go/blob/master/strip_test.go)

**Comment by matthewthomsonnz** [comment]
> I dont suppose you have tested any of the builtin helpers for cleaning html? https://github.com/CloudyKit/jet/wiki/4.-Built-in-functions#escape-helper

None strip tags, and tags appear to be cleaned by default when `{{item.InnerItem.Tagline}}` for example

**Comment by additiverich** [comment]
> tags appear to be cleaned by default

To clarify, < & > get turned into &amp;lt; and &amp;gt; by default so HTML tags get turned into `< br />` in the middle of a sentence in some fields (unless piped through `raw`), which is not what we want here.

**Review by additiverich — APPROVED** [review]

**Pull request merged**